src/main/java/org/zalando/nakadi/plugin/api/authz/EventAuthz.java

```diff
@@ -0,0 +1,15 @@
+package org.zalando.nakadi.plugin.api.authz;
+
+public interface EventAuthz
+{
+    public enum CompatibilityMode {
+        FORWARD, COMPATIBLE, NONE;
+    }
+    public enum CleanupPolicy {
+        DELETE, COMPACT
```

Comment:
The same as the comment above applies for this.

Reply:
Resolved